docs/docsite/rst/user_guide/playbooks_async.rst

```diff
@@ -72,21 +62,12 @@ poll value is set by the ``DEFAULT_POLL_INTERVAL`` setting if you do not specify
   task when run in check mode. See :ref:`check_mode_dry` on how to
   skip a task in check mode.
 
+Run tasks concurrently: poll = 0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Concurrent tasks: poll = 0
---------------------------
-
-When ``poll`` is 0, Ansible will start the task and immediately move on to the next one without waiting for a result.
-
-From the point of view of sequencing this is asynchronous programming: tasks may now run concurrently.
-
-The playbook run will end without checking back on async tasks.
-
-The async tasks will run until they either complete, fail or timeout according to their `async` value.
-
-If you need a synchronization point with a task, register it to obtain its job ID and use the :ref:`async_status <async_status_module>` module to observe it.
+If you want to run multiple tasks in a playbook concurrently, use ``async`` with ``poll`` set to 0. When you set ``poll: 0``, Ansible starts the task and immediately moves on to the next task without waiting for a result. Each async task runs until it either completes, fails or times out (runs longer than its ``async`` value). The playbook run ends without checking back on async tasks.
 
-You may run a task asynchronously by specifying a poll value of 0::
+To run a playbook task asynchronously::
 
```

Comment:
I think extending the timeout is a more common use case than running multiple tasks concurrently, so keeping it closer to the top seems useful. 

If I've understood the functionality correctly, with `poll = 5` Ansible closes the connection, then re-opens it to check for results every five seconds, instead of holding the connection open. So the behavior is asynchronous, even though it isn't running multiple tasks in parallel.

Good point about the titles. I'll play with some options there.